integration-tests/panache-rest-hibernate-orm/src/test/java/io/quarkus/it/panache/rest/common/AbstractBookCrudResourceTest.java

```diff
@@ -83,6 +84,14 @@ void shouldGetAll() {
         assertBook(secondExpectedBook, actualBooks.getJsonObject(1));
     }
 
+    @Test
+    void shouldGetAllWithLimit() {
```

Comment:
Maybe also add tests for `0` and `notAnInt`? Mentioning it because it's easy to get the bytecode wrong (and end up with very obscure errors) when the path has not been tested at all.

Reply:
Yeah I'll add that. But the tests are getting a bit messy. I'll split them up next week as a separate PR to make them cleaner.

Reply:
Sure thing, up to you :)